cmd/server/assets/header.html

```diff
@@ -141,6 +141,12 @@
               {{t $.locale "nav.signing-keys"}}
             </a>
           {{end}}
+          {{if $currentMembership.Can rbac.SettingsRead}}
```

Comment:
Nit, it'd probably be better to just lump this in with the existing "SettingsRead" rbac check on 138